packs/spell-effects/spell-effect-runic-impression.json

```diff
@@ -0,0 +1,162 @@
+{
+    "_id": "Vc97f8ChfcZzprlZ",
+    "img": "icons/magic/symbols/runes-etched-steel-blade.webp",
+    "name": "Spell Effect: Runic Impression",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Runic Impression]</p>\n<p>Your unarmed attacks or weapon gain the benefits of a weapon rune you choose.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "minutes",
+            "value": 1
+        },
+        "level": {
+            "value": 4
+        },
+        "publication": {
+            "license": "OGL",
+            "remaster": false,
+            "title": "Pathfinder Secrets of Magic"
+        },
+        "rules": [
+            {
+                "choices": {
```

Comment:
This may need to instead be:
```
"choices": {
    "attacks": true
}
```

Reply:
If it's attacks true would it not only apply to the single unarmed attack you chose as opposed to all of them? I think I'd sooner do a nested choice between unarmed attack and weapon and adjust the AdjustStrike predicate accordingly

Reply:
I think with weapon it will only allow handwraps? Does it work on regular unarmed attack without it?

Reply:
Also Arodeth has a copy of the spell that needs to have the link added. I don't think you can choose a weapon for an NPC?

Reply:
Only handwraps, so it's either handwraps and all unarmed attacks, or `"attacks": true` and only one unarmed attack. Another Choice Set would do the trick though.

Reply:
I spotted a mistake in the AdjustStrike anyway, so I'll take another crack at it and let you know!